Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocColumnRendererBase.cs

```diff
@@ -155,6 +158,10 @@ protected virtual void RenderTitleCell (BocColumnRenderingContext<TBocColumnDefi
       renderingContext.Writer.RenderEndTag();
     }
 
+    public virtual void AddAttributesToRenderForTitleCell (BocCellAttributeRenderingContext<TBocColumnDefinition> renderingContext, in BocTitleCellRenderArguments arguments)
```

Comment:
Please check: `renderingContext` should also use `in`

Reply:
Since `BocCellAttributeRenderingContext<TBocColumnDefinition>` is just a wrapped rendering context we are introducing another layer of pointers for no reason other than both parameters having an `in` modifier

Reply:
Ah, yes, just one pointer and won't get to be more than one pointer. So we don't need to do any perf optimizations here.